### PR TITLE
Feature/emmy

### PR DESCRIPTION
**Pull request by emmyxth**
# *Name of the PR*

## :recycle: Current situation & Problem
*Describe the current situation (if possible, with an exemplary (or real) code snippet and/or where this is used)*
*Please link any open issue that is addressed with this PR*
Tasked with creating a personal user feature page
## :bulb: Proposed solution
*Describe the solution and how this affects the project and internal structure*
Created user feature page with buttons redirecting to external links
## :gear: Release Notes 
*Add a summary of the feature and possible migration guides if this is a breaking change so this section can be added to the release notes.*
*Include code snippets that provide examples of the feature implemented if it appends or changes the public interface.*
Single page describing aspects of personality, and buttons redirecting to external links
## :heavy_plus_sign: Additional Information
*Provide some additional information if possible*

### Related PRs
*Reference the related PRs*

### Testing
*Are there tests included? If yes, which situations are tested, and which corner cases are missing?*
Yes, I test the buttons on my page to see if they work.
### Reviewer Nudging
*Where should the reviewer start? Where is a good entry point?*
Starting from member views and clicking on my name.
### Code of Conduct & Contributing Guidelines 

By submitting creating this pull request, you agree to follow our [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md):
- [ x] I agree to follow the [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: CS342Application/Assets.xcassets/mycolor.colorset/Contents.json
Comment (PSchmiedmayer) [inline CS342Application/Assets.xcassets/mycolor.colorset/Contents.json]: I have not seen that the color is used in your views. You might want to remove it if it is not used.

File: CS342MemberViews/Sources/EmmyThamakaison/BubbleView.swift
Comment (PSchmiedmayer) [inline CS342MemberViews/Sources/EmmyThamakaison/BubbleView.swift:17]: Very cool that you have separated the facts out into a model type that you use in your view, very good solution! 👍

File: CS342MemberViews/Sources/EmmyThamakaison/LinkedIn.swift
Comment (PSchmiedmayer) [inline CS342MemberViews/Sources/EmmyThamakaison/LinkedIn.swift:21]: You have a LinkedIn icon in your Resources folder that you might want to remove as it is not used.

**Review by PSchmiedmayer — APPROVED** [review]
Hi @emmyxth 

Thank you for your PR and your nice view!
I have added some comments on how to improve your code in the PR review.

The next step would be 
1. ... to fix the remaining SwiftLint errors locally by installing SwiftLint or going through the error messages in the PR diff here on GitHub. You can find out more on how to install SwiftLint on your machine here: https://github.com/realm/SwiftLint
2. ... to also fix the REUSE errors by adding a license to the `CS342MemberViews/Sources/CaitlinKunchur/Resources/CaitlinKunchur-bio.md` file or adding a `CaitlinKunchur-bio.md.license` file in the same folder the appropriate copyright statement.

I have also merged the latest main state in your branch to make merging the PR easy.
Be sure that you `git pull` the latest state to your local version if you accept any of the suggestions here on GitHub and get the merge conflict to your local branch 🚀

**Comment by PSchmiedmayer** [comment]
@emmyxth A short reminder to improve your Assignment 1 submission. We would like to merge it in the next few days so you can concentrate on your project assignments 👍

Feel free to tag me if you have any questions. 
Please also tag me if you want me to take an other look at the PR.
I am also happy to help you with any merge conflicts that you might have to speed up the merge so that you can focus on the content of the exercise!

Feel free also to approach me after class today so we can take a look at the PR together and merge it directly if you have incorporated my feedback by then 🚀

**Comment by emmyxth** [comment]
Hi @PSchmiedmayer , I've been trying to commit my changes after last week, and I'm not sure why this error message keeps  popping up:
Screenshot 2566-02-01 at 14 49 50
Would it be possible to work this out with you some time tomorrow during/after class?

**Comment by PSchmiedmayer** [comment]
Hi @emmyxth,

The error message looks like you have not yet added the current changes to your staging area to commit them.

You can learn more about git and what git add + commit does here: https://www.atlassian.com/git/tutorials/saving-changes

Also as noted in my comment of my review above (and also noted in the message you get in the terminal), you also need to `git pull` the latest state to your local version if you accept any of the suggestions here on GitHub and get the merge conflict to your local branch.
You can do this after you committed your latest changes. 
The git pull might result in a merge that you will need to resolve. Git let you know if that this the case.

You can learn more about git and what git pull does here: https://www.atlassian.com/git/tutorials/syncing/git-pull

We can also take a look at this together tomorrow after the class.

**Comment by emmyxth** [comment]
Okay, thank you for dropping those resources! I think I managed to merge the branch onto the main branch. Am I now waiting for your approval or is there anything else that I need to do?

**Comment by PSchmiedmayer** [comment]
@emmyxth I will take a look at the PR and will let you know about the next steps 👍

**Comment by emmyxth** [comment]
Thank you, Paul! 🙏🏼

**Review by PSchmiedmayer — APPROVED** [review]

**Comment by PSchmiedmayer** [comment]
@emmyxth I took a look at your PR, merged the main branch into your branch and will merge the PR. Thank you for addressing the comments. I will merge the PR once all the test have passed 🚀

**Pull request merged**
